hazelcast/cluster.py

```diff
@@ -58,9 +67,11 @@ class ClusterService(object):
     """
 
     def __init__(self, internal_cluster_service):
+        # type: (Any) -> None
         self._service = internal_cluster_service
 
     def add_listener(self, member_added=None, member_removed=None, fire_for_existing=False):
+        # type: (function, function, bool) -> str
```

Comment:
We could be more specific about the `function` type here, what are the parameters or the return type?